src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -651,7 +643,6 @@ private void remove(Resident resident) {
 					if ((assistant != resident) && (resident.hasTownRank("assistant"))) {
 						try {
 							setMayor(assistant);
```

Comment:
I don't know why this continue was removed. Would this not end up making each assistant mayor until the last one is finally made mayor?

Reply:
The readability of the for-loop is difficult because the brackets are missing from them (should add the brackets around the for-loop), but the continue was redundant as the try-catch block is the only thing executed in the loop. Applies for the same comment below.

Reply:
The continue was probably a typo (or maybe it wasn't at a different time,) and has since been replaced with breaks.

Reply:
I mean I can add brackets 😄